sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/PostgresqlDatabaseIT.java

```diff
@@ -7,12 +7,12 @@
 import io.quarkus.test.services.QuarkusApplication;
 
 @QuarkusScenario
-public class Postgresql13DatabaseIT extends AbstractReactiveDatabaseIT {
+public class PostgresqlDatabaseIT extends AbstractReactiveDatabaseIT {
```

Comment:
Why this rename? There is also `OpenShiftPostgresql10IT.java`. I would really keep the naming as is in this PR.

If you have proposal for DB related TestCases renaming, please bring it up on Monday meeting.

Reply:
OK, looking more into the code and it probably makes sense to do it this way, it keeps consistency with other tests naming in that module.

Created trello task to discuss the generic naming topic for DB related test cases.